src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1825,17 +1828,19 @@ Made with 💗 by Microsoft and the PowerToys community.</value>
   </data>
   <data name="FancyZones_MoveWindowBasedOnRelativePosition_Accessible.[using:Microsoft.UI.Xaml.Automation]AutomationProperties.Name" xml:space="preserve">
     <value>Windows key + Up, down, left or right arrow key to move windows based on relative position</value>
+	  <comment>{Locked="Windows"}</comment>
   </data>
   <data name="FancyZones_MoveWindowLeftRightBasedOnZoneIndex_Accessible.[using:Microsoft.UI.Xaml.Automation]AutomationProperties.Name" xml:space="preserve">
     <value>Windows key + Left or right arrow keys to move windows based on zone index</value>
+	  <comment>{Locked="Windows"}</comment>
   </data>
   <data name="FancyZones_MoveWindowBasedOnRelativePosition_Description.Text" xml:space="preserve">
     <value>Windows key +    or </value>
-    <comment>Do not loc the icons (hex numbers)</comment>
+    <comment>{Locked="Windows key +   ",""</comment>
   </data>
   <data name="FancyZones_MoveWindowLeftRightBasedOnZoneIndex_Description.Text" xml:space="preserve">
     <value>Windows key +  or </value>
-    <comment>Do not loc the icons (hex numbers)</comment>
+    <comment>{Locked}</comment>
```

Comment:
the previous one, you locked only the key names. This one, you locked the whole thing. This will result in ` or ` not being translated.

You should lock only the key names.

Reply:
good catch, think i missed the or